canopen/objectdictionary/eds.py

```diff
@@ -45,7 +45,7 @@ def import_eds(source, node_id):
     if eds.has_section("Comments"):
         linecount = int(eds.get("Comments", "Lines"), 0)
         od.comments = '\n'.join([
-            eds.get("Comments", "Line%i" % line)
+            eds.get("Comments", f"Line{line}")
```

Comment:
This might break when line is not an integer. Previously, it would raise an exception. The new code happily generates invalid EDS files. So better to put the format specifier also in the F-string?

Reply:
`line` cannot become anything else than int as the line below is `for line in range(1, ...)`.

And I would be worried if the assurance of generating proper EDS hinges on a specific format specifier. If that would be the case, I'd say we need something more explicit like a test and raising an error. ;)